### PR TITLE
Removed 'enumeration not handled' warnings

### PR DESCRIPTION
**Pull request by simon987**

### CODE REVIEW

File: common/game-state-queue.cpp [hunk 1]
```diff
@@ -112,30 +112,57 @@ namespace tec {
...
+				case proto::Component::kVelocity:
+				{
+					Velocity vel;
+					vel.In(comp);
+					this->interpolated_state.velocities[entity_id] = vel;
+					this->base_state.velocities[entity_id] = vel;
+				}
+					break;
+				case proto::Component::kRenderable:
+					break;
```
Comment (adam4813) [inline common/game-state-queue.cpp]: Most of these breaks can be removed as the cases will falldown till exit regardless.

File: common/physics-system.cpp [hunk 2]
```diff
@@ -335,15 +335,45 @@ namespace tec {
...
+				{
+					CollisionBody* collision_body = new CollisionBody();
+					collision_body->In(comp);
+					CollisionBodyMap::Set(entity_id, collision_body);
+					collision_body->entity_id = entity_id;
+					AddRigidBody(collision_body);
+				}
+                    break;
+                case proto::Component::kRenderable:
+                    break;
```
Comment (adam4813) [inline common/physics-system.cpp]: Most of these breaks can be removed as the cases will falldown till exit regardless.

File: client/resources/pixel-buffer.cpp [hunk 3]
```diff
@@ -86,6 +86,12 @@ namespace tec {
 					if (b < 3) std::fputc(p[i], stderr); // only output RGB
 				}
 				break;
+			case ImageColorMode::MONOCHROME:
+				break;
```
Comment (adam4813) [inline client/resources/pixel-buffer.cpp]: Most of these breaks can be removed as the cases will falldown till exit regardless.

File: client/resources/pixel-buffer.cpp [hunk 4]
```diff
@@ -107,6 +113,13 @@ namespace tec {
 					file.write((char*)(p + i), 3); // only output RGB
 				}
 				break;
+			case ImageColorMode::MONOCHROME:
+				break;
```
Comment (adam4813) [inline client/resources/pixel-buffer.cpp]: Most of these breaks can be removed as the cases will falldown till exit regardless.

File: client/render-system.cpp [hunk 5]
```diff
@@ -449,14 +449,44 @@ namespace tec {
...
+					renderable->In(comp);
 
-				RenderableMap::Set(entity_id, renderable);
-			}
-			break;
+					RenderableMap::Set(entity_id, renderable);
+				}
+					break;
+				case proto::Component::kPosition:
+					break;
```
Comment (adam4813) [inline client/render-system.cpp]: Most of these breaks can be removed as the cases will falldown till exit regardless.

**Comment by milesrout** [comment]
Is it actually okay for these cases to be ignored?

**Comment by adam4813** [comment]
This is building, but for some reason nothing is showing up when I load. I am investigating further as it could be a cache issue.

**Pull request merged**